src/main/java/com/example/ecom/controller/AreaController.java

```diff
@@ -40,12 +57,15 @@ public void delete(@PathVariable Long id) {
         areaRepository.deleteById(id);
     }
 
+    // Update an area
     @PutMapping("/{id}")
-    public void update(@PathVariable Long id, @RequestBody Area area){
+    public Area update(@PathVariable Long id, @RequestBody String areaName){
```

Comment:
I want the request body as Object since I might add something more than a name like inactive flag.